src/components/composer/composer/abstract_composer_store.ts

```diff
@@ -94,6 +99,7 @@ export abstract class AbstractComposerStore extends SpreadsheetStore {
   protected initialContent: string | undefined = "";
   private colorIndexByRange: { [xc: string]: number } = {};
   private autoComplete: Store<AutoCompleteStore> = new AutoCompleteStore(this.get);
+  private hasSelectedAProposal: boolean = false;
```

Comment:
This kinda sucks ... but I don't have a better idea.

The problem is if we open a composer on a formula like `=MyNamedRange`. The autocomplete dropdown is open, and `MyNamedRange` is selected. If we press enter, `provider.selectProposal` will be called, do nothing, the dropdown will be closed ... and the promptly re-opened since nothing changed, so pressing enter will do absolutely nothing and we cannot close the composer.

And we don't want to hide the dropdown if `this._currentContent === this.initialContent` otherwise we don't have functions auto-complete either ...